README.md

```diff
@@ -1,20 +1,118 @@
 # Welcome to Fetsy Seller
 
 Get started
-- Run `git clone https://github.com/rpt24sourcandy/fetsySeller.git`
+- Run `git clone https://github.com/rpt24baklannoli/fetsySeller.git`
 - Run `npm install`
-- Run `npm run db:setup` and `npm run db:seed` (TODO: combine commands)
+- Run `npm run db:setup` and `npm run db:seed`
+- Run `npm run dev:react`
 - Run `npm start`
 
 Endpoints
-- `/items/:item_id/seller` response will be seller information for given item id
-    {
-      "item_id": 1
-      "seller_id": 1,
-      "seller_name": "Forest.Swaniawski14",
-      "seller_rating": 3,
-      "total_sales": 893,
-      "seller_city": "North Gordon",
-      "seller_state": "Oklahoma",
-      "on_etsy_since": 2019,
-    }
+- `/items/:item_id/seller` response will be seller information for given item_id
+```
+{
+  "seller_id": 1,
+  "seller_rating": 4,
+  "total_sales": 201,
+  "seller_name": "Annamae.Willms37",
+  "seller_city": "Ellatown",
+  "seller_state": "New York",
+  "on_etsy_since": 2015,
+  "item_id": 1
+}
+```
+
+<a name="crud"></a>
+# CRUD Documentation
+
+This service supports full CRUD functionality, available for testing with Postman. Not all functionality is utilized by the front end.
+
+## Summary
+| Methods  | URL Endpoints | Actions |
+| :------- | :------------ | :------ |
+| GET | /items/:item_id/seller  | Retrieve seller info for a given item |
+| POST | /items/:item_id/seller | Add a new seller for a given item |
+| PUT | /items/:item_id/seller | Update the seller info for a given item |
+| DELETE | /items/:item_id/seller | Delete seller info for a given item |
+
+## Read (GET) Seller Information
+> Example use case: retrieve seller information for an item
+
+#### Input
+```sh
+Endpoint: '/items/:item_id/seller'
+```
+#### Output
+```sh
+Sample Response:
+{
+  "seller_id": 1,
+  "seller_rating": 4,
+  "total_sales": 201,
+  "seller_name": "Annamae.Willms37",
+  "seller_city": "Ellatown",
+  "seller_state": "New York",
+  "on_etsy_since": 2015,
+  "item_id": 1
+}
+```
+
+## Create (POST) A New Seller
+> Example use case: add a new seller to the database for an item
+
+#### Input
+> Note: item_id auto increments and is not required in the request body.
+
+```sh
+Endpoint: '/items/:item_id/seller'
+
+Request Body Example:
+{
+  "seller_rating": 3,
+  "total_sales": 25,
+  "seller_name": Ashley Kelly,
+  "seller_city": Falls Church,
+  "seller_state": Georgia,
+  "on_etsy_since": 1995
+}
+```
+#### Output
+> Success: 200 status code, and a response of "Successfully added new seller"
```

Comment:
Double check your status code listed here vs. what's in your actual output. I think you had this as 201?